sample-ms/src/main/java/net/trajano/ms/example/HelloResource.java

```diff
@@ -168,6 +169,18 @@ public Response maven() {
         }
     }
 
+    @SuppressWarnings("null")
+    @ApiOperation(value = "throw null pointer exception")
+    @GET
+    @Path("/null")
+    @Produces(MediaType.APPLICATION_JSON)
+    public JsonObject nullOp() {
+
+        final JsonObject ret = null;
+        ret.add("Hello", new JsonPrimitive("world"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "ret" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)